cli_tools/google-osconfig-agent/config/config.go

```diff
@@ -27,8 +32,17 @@ import (
 const (
 	// InstanceMetadata is the instance metadata URL.
 	InstanceMetadata = "http://metadata.google.internal/computeMetadata/v1/instance"
-	// ReportURL is where OS configurations are written in guest attributes.
+	// ReportURL is the guest attributes endpoint.
 	ReportURL = InstanceMetadata + "/guest-attributes"
+
+	googetRepoFilePath = "C:/ProgramData/GooGet/repos/google_osconfig.repo"
+	zypperRepoFilePath = "/etc/zypp/repos.d/google_osconfig.repo"
+	yumRepoFilePath    = "/etc/yum/repos.d/google_osconfig.repo"
+	aptRepoFilePath    = "/etc/apt/sources.list.d/google_osconfig.list"
+
+	osInventoryEnabled = false
+	osPackageEnabled   = false
+	osPatchEnabled     = false
```

Comment:
minor: can you add `default` to the names to differentiate them from other variables with the same name?